recipe/meta.yaml

```diff
@@ -17,34 +17,31 @@ build:
 requirements:
   host:
     - python
-    - flit
     - pip
-    - setuptools >=40.8.0
+    - hatchling
     - tornado
+    - setuptools
     - wheel
   run:
     - python
     - ptyprocess        # [not win]
     - pywinpty >=1.1.0  # [win]
-    - tornado >=4
+    - tornado >=6.1.0
 
 test:
   requires:
     - pip
     - posix  # [win]
-    - pytest
+    - pytest >=6.0
     - pytest-cov
+    - pytest-timeout
   imports:
     - terminado
   commands:
     # see run_test.py for more
     - pip check
-  # Downstream packages are jupyter_server and notebook
-  # Testing them to verify that they will succeed
-  downstreams:
```

Comment:
Just curious why you removed downstreams?

Reply:
They don't appear on conda-forge, so I assumed they were no longer needed. Is that something we need to keep? https://github.com/conda-forge/terminado-feedstock/blob/main/recipe/meta.yaml

Reply:
notebook is already have the Python 3.10 support. You can remove the constraint. 

Before merging I would recommend to comment the downstreams section.